### PR TITLE
basic structure

### PR DESCRIPTION
**Pull request by shahira-sadat**
Hi, Thanks for your review.

Following is the list of activities done for this PR 👇🏻:

1. Create an npm project with webpack.
2. Use Gitflow
3. Write plain HTML markup with minimum styling
4. Create modules for generating score `data.js`  and adding score `score.js`
5. Use ES6 modules, with import and export.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
     "eslint": "^7.32.0",
     "eslint-config-airbnb-base": "^14.2.1",
     "eslint-plugin-import": "^2.26.0",
-    "hint": "^6.2.0",
+    "hint": "^2.0.0",
```
Comment (CollinsTatang) [inline package.json]: Hello @shahira-sadat 
Great job so far implementing the project requirements.

- [x] Your app throws an error when I try to start it with webpack `npm run start`. Indicating that you are missing some `package.json` dependencies.
solutio
### **SOLUTION**
You can install the missing dependencies by running this command on your terminal from your project root directory.
`npm install --save-dev babel-loader @babel/core @babel/preset-env webpack`

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @shahira-sadat, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required changes 
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job.
_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json (hunk 1 above)
Comment (shahira-sadat) [reply package.json]: Done

**Review by prolajumokeoni — APPROVED** [review]
Hi @shahira-sadat, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Git flow was used correctly 🚀 
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**